mlir/lib/Dialect/SCF/TransformOps/SCFTransformOps.cpp

```diff
@@ -476,21 +478,40 @@ static bool isForallWithIdenticalConfiguration(Operation *target,
          targetOp.getMapping() == sourceOp.getMapping();
 }
 
-/// Fuse `target` into `source` assuming they are siblings and indepndent.
-/// TODO: Add fusion for more operations. Currently, we handle only scf.forall.
+static bool isForWithIdenticalConfiguration(Operation *target,
```

Comment:
Nit: please add documentation to free functions.

Reply:
Done.